core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpActivity.kt

```diff
@@ -31,14 +31,16 @@ import kotlinx.android.synthetic.main.activity_help.activity_help_feedback_image
 import kotlinx.android.synthetic.main.activity_help.activity_help_feedback_text_view
 import kotlinx.android.synthetic.main.activity_help.activity_help_recycler_view
 import kotlinx.android.synthetic.main.layout_toolbar.toolbar
-import java.util.HashMap
 
 class HelpActivity : BaseActivity() {
   private val titleDescriptionMap by lazy {
     listOf(
       R.string.help_2 to R.array.description_help_2,
       R.string.help_5 to R.array.description_help_5
-    )
+    ).associate {
```

Comment:
Sorry I keep nitpicking this function.

We can also use a [destructuring declaration](https://kotlinlang.org/docs/reference/multi-declarations.html#destructuring-in-lambdas-since-11) in the associate lambda because `it.first/second` aren't very descriptive